src/TesApi.Web/scripts/config-n-gpu-apt.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+trap "echo Error trapped; exit 0" ERR
+# set -e will cause any error to exit the script
+set -e
+curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
```

Comment:
can you add a comment with a link to the NVIDIA docs if you have one?

Reply:
Done